go.mod

```diff
@@ -4,13 +4,40 @@ go 1.16
 
 require (
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
+	github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6 // indirect
```

Comment:
It seems that no new packages are used in the api, where do these new requirements come from?

Reply:
Emmm...maybe i'll should use go.mod in previous version...

Reply:
I'll try using go.mod in previous version.